torchtitan/config_manager.py

```diff
@@ -450,6 +450,13 @@ class Float8:
     Example: --float8.filter_fqns "attention.wq,attention.wk,attention.wv,output"
     """
 
+    emulate: bool = False
+    """
+    If True, emulation is used instead of hardware accelerated gemm. This is for test purpose only,
+    as the current CI does have sm_90 capability, required by Float8.
+    Not compatible with torch.compile.
```

Comment:
```suggestion
    If True, emulation is used instead of hardware accelerated gemm. This is for test purpose only,
    as the current CI does not have sm_89 capability, required by Float8.
```

Reply:
could you make this update to the doc string? otherwise it seems inaccurate
```suggestion
    If True, emulation is used instead of hardware accelerated gemm. This is for test purpose only,
    as the current CI does not have sm_89 capability, required by Float8.
```